stylesheets/_component.datatables.scss

```diff
@@ -46,15 +46,26 @@
 }
 // scss-lint:enable SelectorFormat
 
-.dt-buttons {
-    border-right: 1px solid #ccc;
+.dt-buttons > a {
+    float: left;
+    margin-top: 11px;
+    padding-right: 10px;
+    border-right: 1px solid color(border);
+}
+
+a.dt-button {
+    margin-right: 10px;
+}
+
+.dt-buttons > button {
```

Comment:
Could this selector be merged with `.dt-buttons > a`? Looks to have the same rules other than a different padding.

For example:
```
.dt-buttons > a,
.dt-buttons > button {
```